ember-app/app/initializers/sockets.js

```diff
@@ -36,6 +37,7 @@ export function initialize(container, application){
         return callback;
       },
       unsubscribe: function(channel, callback) {
+        if(channel == "null") { return Ember.K; }
```

Comment:
Is this really supposed to guard against the string `"null"` rather than a falsey value of `channel`? (Same question above)